doc/user/content/releases/v0.48.md

```diff
@@ -59,8 +59,17 @@ shipping in v0.48 -— so mentioning it here."
 
 * Stop silently ignoring `NULL` keys in sources using `ENVELOPE UPSERT` {{% gh 6350 %}}.
   The new behavior is to throw an error when trying to query the source. To
-  recover an errored source, the upstream system must emit a record with a `NULL`
-  value and a `NULL` key.
+  recover an errored source, the upstream Kafka broker must emit a record with
```

Comment:
Not sure what "the upstream Kafka broker must emit" means, exactly. I'd think you "have to write a record with a `NULL` value and a `NULL` key to the topic, to force a retraction". wdyt?

Same is true for the other occurrence of this text block in the Kafka docs below.